llvm/test/Instrumentation/MemorySanitizer/SPIRV/check_large_access_size.ll

```diff
@@ -1,19 +0,0 @@
-; RUN: opt < %s -passes=msan -msan-instrumentation-with-call-threshold=0 -msan-eager-checks=1 -S | FileCheck %s
```

Comment:
This test is removed because we remove "tls" and needn't check function parameter.